diff -crB gcc-4.1.2-orig/gcc/d/d-lang.cc gcc-4.1.2/gcc/d/d-lang.cc
*** gcc-4.1.2-orig/gcc/d/d-lang.cc	2007-08-22 02:16:39.000000000 +0200
--- gcc-4.1.2/gcc/d/d-lang.cc	2009-04-06 17:55:06.000000000 +0200
***************
*** 185,191 ****
  #undef D_PHOBOS_TARGET_DIR
  #define D_PHOBOS_TARGET_DIR (cygwin_d_target_dir)
  
! const char * cygwin_d_os_versym = D_OS_VERSYM;
  #undef D_OS_VERSYM
  #define D_OS_VERSYM cygwin_d_os_versym
  
--- 185,191 ----
  #undef D_PHOBOS_TARGET_DIR
  #define D_PHOBOS_TARGET_DIR (cygwin_d_target_dir)
  
! const char * cygwin_d_os_versym = "MyOS";
  #undef D_OS_VERSYM
  #define D_OS_VERSYM cygwin_d_os_versym
  

