diff -rupN qemu-1.0.orig/ui/cocoa.m qemu-1.0/ui/cocoa.m
--- qemu-1.0.orig/ui/cocoa.m	2011-12-06 18:33:47.000000000 +0100
+++ qemu-1.0/ui/cocoa.m	2011-12-06 18:36:08.000000000 +0100
@@ -22,9 +22,6 @@
  * THE SOFTWARE.
  */
 
-#import <Cocoa/Cocoa.h>
-#include <crt_externs.h>
-
 #include "qemu-common.h"
 #include "console.h"
 #include "sysemu.h"
@@ -36,6 +33,9 @@
 #define MAC_OS_X_VERSION_10_5 1050
 #endif
 
+#define _UINT16
+#import <Cocoa/Cocoa.h>
+#include <crt_externs.h>
 
 //#define DEBUG
 
